packages/core/src/Banner.js

```diff
@@ -73,9 +73,9 @@ const Banner = props => {
       color={color}
     >
       <Flex justifyContent="space-between" alignItems="flex-start">
-        {!!icon && !!props.showIcon && (
-          <Icon name={icon} mr={2} size={24} mt="-2px" />
-        )}
+        {!!icon &&
+          !!props.showIcon &&
+          React.cloneElement(props.icon, { size: 24, mt: '-2px', mr: 2 })}
```

Comment:
I did the same thing in my PR, we might get a few conflicts but I'll resolve them, it's my bad